tests/test_explicit_keys.py

```diff
@@ -23,23 +23,23 @@ class Delta(Charlie):
 
 
 def test_tracked_descendants():
```

Comment:
(Optional) Could be updated to `test_progeny` to match the property method name. Generally on the pbm codebase we try to keep the test names something like `test_method_name` where `method_name` is the method being tested.

Reply:
Yeah, this test name should be updated.